packages/account-abstraction/src/Provider.ts

```diff
@@ -22,7 +22,14 @@ export async function newProvider(
   const entryPoint = EntryPoint__factory.connect(config.entryPointAddress, originalProvider)
   // Initial SimpleWallet instance is not deployed and exists just for the interface
   // const simpleWalletDeployer = await DeterministicDeployer.deploy(SimpleAccountDeployer__factory.bytecode)
+  const httpRpcClient = new HttpRpcClient(
```

Comment:
eventually initialising bundler client won't be tied with needing to pass EP address and DappAPIKey etc..